### DISCUSSION TITLE
Is there any possibility to pass password for VNC Server?

### DISCUSSION THREAD

**Discussion by ChyngyzMonokbaev** [roerohan/react-vnc#34] [Q&A] [answered]
How to add password, if VNC server requires it?

**Comment by roerohan** [comment, marked as answer]
@monokbaev , sorry for the late response.

https://github.com/roerohan/react-vnc/blob/01afe07b5801087fbf11fec366896c980986ac54/src/lib/VncScreen.tsx#L95-L98

I haven't tried it with a password yet, but this part should ensure that you get a prompt on your browser in case the VNC server requires a password.
